mypy/plugins/common.py

```diff
@@ -55,7 +58,7 @@ def _get_argument(call: CallExpr, name: str) -> Optional[Expression]:
     callee_node = call.callee.node
     if (isinstance(callee_node, (Var, SYMBOL_FUNCBASE_TYPES))
             and callee_node.type):
-        callee_node_type = callee_node.type
+        callee_node_type = get_proper_type(callee_node.type)
```

Comment:
Since this change may affect plugins, it needs to be documented in plugin docs and the plugins changelog issue before the next mypy release.

Reply:
Yes, I actually added this expansion to minimize possible breakages (at least few of our plugins used `isinstance()` with these types).